src/ui/wxWidgets/I18N/Makefile

```diff
@@ -39,7 +39,7 @@ $(POS) : $(POT)
 
 mos/%/LC_MESSAGES/pwsafe.mo : pos/pwsafe_%.po
 	@mkdir -p $(dir $@)
-# do not remove wanted entries:	@msgfmt --check --output-file=$@ $^
+# "--check" option would remove potential used entries, so do not add it to the following line
```

Comment:
Could someone please explain to me what effect this actually has on the translations in practice?
Would the resulting effect have an advantage over the missing checks, especially the [format checks](https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html)?

Reply:
We have several items in MacOS that are not found in the source tree. Using the clean option always comment out those items, not aligned to a source of line, but needed for text translations on MacOS standard menu.

Reply:
At least in my opinion, the "check-format" option is very valuable if it is not included elsewhere on the toolchain.

Reply:
There is no missing format check, but the entries defined in the pwsage_xx.po not related to a source line, e.g. those for MacOS standard menu, are comment out when the check option is present.

Reply:
I think I have to change back. The problems occurs more when the po file is newly generated. I will update the makefile soon.